internal/commands/root.go

```diff
@@ -91,12 +91,14 @@ func (o *parametersOptions) addFlags(flags *pflag.FlagSet) {
 type credentialOptions struct {
 	targetContext    string
 	credentialsets   []string
+	credentials      []string
 	sendRegistryAuth bool
 }
 
 func (o *credentialOptions) addFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.targetContext, "target-context", "", "Context on which the application is installed (default: <current-context>)")
 	flags.StringArrayVar(&o.credentialsets, "credential-set", []string{}, "Use a YAML file containing a credential set or a credential set present in the credential store")
+	flags.StringArrayVar(&o.credentials, "credential", nil, "Add a single credential, additive ontop of any --credential-set used")
```

Comment:
I wonder if we should support the CSV style format? 🤔 
```sh
$ docker app install --credential cred1=foo,cred2=bar --credential cred3=baz
```

Reply:
Can that handle a credential with a literal `,` in it?

If there is a library call (in the CLI or elsewhere) which I can use which will Just Work the same as everywhere else then I am happy to do that, but I don't really want to write bespoke code here to do that.

Reply:
Oh right... ok then 👍